### DISCUSSION TITLE
How can I focus markers?

### DISCUSSION THREAD

**Discussion by liamskydamien** [react-native-maps/react-native-maps#4927] [Q&A]
Hello everyone,
Im currently integrating a slider component to the Map. 
Im showcasing markers on the map and for each marker I have a card in my slider.
If I swipe through the slider the corresponding marker to the active card should be focused the map should zoom to it.
I see in the README.md that this is possible, but I dont understand how I can implement this feature.
Can someone help?

**Comment by ya-nsh** [comment]
Hi, I have the exact use case. Have you found any solution for this?

**Reply by liamskydamien** [reply]
I found a way by implementing a ref through which I can trigger the "zoom to" function. Unfortunately you need to manage a list of markers between both components the slider and the map, which can make it quite difficult and lead to some weird zoom behavior. So it's important to have a good state management
